website_event_compassion/models/event_registration.py

```diff
@@ -604,6 +611,7 @@ def prepare_group_visit_payment(self):
                 'name': product.name,
                 'product_id': product.id,
                 'account_analytic_id': event.analytic_id.id,
+                'payment_mode_id': mode_pay_bvr.id,
```

Comment:
This is invoice line vals and invoice line doesn't have a `payment_mode_id` field. You can thus remove this line. It should only be included in invoice values.